C_routing.md

```diff
@@ -778,113 +778,50 @@ end
 
-We define sockets in the router with a block, a path to the socket's mount point, and (optionally) the name of our application to fully qualify our channel module name. This should look familiar after seeing our scope definitions above.
-
-Here's what that looks like in our router file:
+We mount socket handlers in our endpoint, who handle authentication callbacks and channel routes.
 
 ```elixir
-defmodule HelloPhoenix.Router do
-  use HelloPhoenix.Web, :router
+defmodule HelloPhoenix.Endpoint do
+  use Phoenix.Endpoint
 
-  socket "/ws", HelloPhoenix do
-  end
+  socket "/socket", HelloPhoenix.UserSocket
   ...
 end
 ```
 
-Next, we need to use the `channel/3` macro to match a topic to the channel which will handle its requests. If we have a channel module called `RoomChannel` and a topic called `"rooms:*"`, the code to do this is straightforward.
+Next, we need to open our `web/channels/user_socket.ex` file and use the `channel/3` macro to define our channel routes. The routes will match a topic pattern to a channel to handle events. If we have a channel module called `RoomChannel` and a topic called `"rooms:*"`, the code to do this is straightforward.
 
 ```elixir
-defmodule HelloPhoenix.Router do
-  use HelloPhoenix.Web, :router
+defmodule HelloPhoenix.UserSocket do
+  use Phoenix.Socket
 
-  socket "/ws", HelloPhoenix do
-    channel "rooms:*", RoomChannel
-  end
-  . . .
+  channel "rooms:*", HelloPhoenix.RoomChannel
+  ...
 end
 ```
 
 Topics are just string identifiers. The form we are using here is a convention which allows us to define topics and subtopics in the same string - "topic:subtopic". The `*` is a wildcard character which allows us to match on any subtopic, so `"rooms:lobby"` and `"rooms:kitchen"` would both match this route.
 
-Now that we have a channel route defined, let's see what `$ mix phoenix.routes` shows us.
-
-```console
-$ mix phoenix.routes
-. . .
-web_socket_path  GET   /ws       Phoenix.Transports.WebSocket.upgrade/2
-web_socket_path  POST  /ws       Phoenix.Transports.WebSocket.upgrade/2
-long_poller_path  GET   /ws/poll  Phoenix.Transports.LongPoller.poll/2
-long_poller_path  POST  /ws/poll  Phoenix.Transports.LongPoller.publish/2
-```
-
-Notice that our socket definition expands out to four paths with two separate transport mechanisms - WebSockets and LongPolling. If we wanted to make sure that our channel is handled by only one type of transport, we could specify that using the `via` option, like this.
+Phoenix abstracts the socket transport layer and includes two transport out of the box - WebSockets and Long-Polling. If we wanted to make sure that our channel is handled by only one type of transport, we could specify that using the `via` option, like this.
```

Comment:
Maybe change "two transport" to "two transport mechanisms" or just "two transports"?